tests/cli/test_cli_set_family.py

```diff
@@ -0,0 +1,152 @@
+
+    result = invoke_cli(
+        ['--database', db_uri, 'set', 'family',
+         '--priority', priority, name])
+
+    # THEN then it should have been set
+    assert result.exit_code == 0
+    assert disk_store.Family.query.count() == 1
+    assert disk_store.Family.query.first().priority_human == priority
+
+
+def add_panel(disk_store, panel_id='panel_test', customer_id='cust_test'):
+    """utility function to set a panel to use in tests"""
+    customer = ensure_customer(disk_store, customer_id)
+    panel = disk_store.add_panel(customer=customer, name=panel_id, abbrev=panel_id,
+                                 version=1.0,
+                                 date=datetime.now(), genes=1)
+    disk_store.add_commit(panel)
+    return panel_id
+
+
+def add_application(disk_store, application_tag='dummy_tag'):
+    """utility function to set an application to use in tests"""
+    application = disk_store.add_application(tag=application_tag, category='wgs',
+                                             description='dummy_description')
+    disk_store.add_commit(application)
+    prices = {'standard': 10, 'priority': 20, 'express': 30, 'research': 5}
+    version = disk_store.add_version(application, 1, valid_from=datetime.now(),
+                                     prices=prices)
+
+    disk_store.add_commit(version)
+    return application_tag
+
+
+def ensure_application_version(disk_store, application_tag='dummy_tag'):
+    """utility function to return existing or create application version for tests"""
+    application = disk_store.application(tag=application_tag)
+    if not application:
+        application = disk_store.add_application(tag=application_tag, category='wgs',
+                                                 description='dummy_description')
+        disk_store.add_commit(application)
+
+    prices = {'standard': 10, 'priority': 20, 'express': 30, 'research': 5}
+    version = disk_store.application_version(application, 1)
+    if not version:
+        version = disk_store.add_version(application, 1, valid_from=datetime.now(),
+                                         prices=prices)
+
+        disk_store.add_commit(version)
+    return version.id
+
+
+def ensure_customer(disk_store, customer_id='cust_test'):
+    """utility function to return existing or create customer for tests"""
+    customer_group = disk_store.customer_group('dummy_group')
+    if not customer_group:
+        customer_group = disk_store.add_customer_group('dummy_group', 'dummy group')
+
+        customer = disk_store.add_customer(internal_id=customer_id, name="Test Customer",
+                                           scout_access=False, customer_group=customer_group,
+                                           invoice_address='dummy_address',
+                                           invoice_reference='dummy_reference')
+        disk_store.add_commit(customer)
+    customer = disk_store.customer(customer_id)
+    return customer
```

Comment:
this is the only `ensure_` function that returns a model object instead of a string. Suggest to return string instead of object for consistency's sake. What do you think?

Reply:
fixed, return only objects